form.go

```diff
@@ -521,7 +524,7 @@ func (f *Form) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		}
 		f.selector.Range(func(_ int, group *Group) bool {
 			if group.fullHeight() > msg.Height {
-				group.WithHeight(msg.Height)
+				group.WithHeight(msg.Height - 1)
```

Comment:
this accounts for the help menu at the bottom of the window taking up one row. Can add a const for this 1 for legibility if you'd prefer